Extending/FileSystemProviders/Azure-Blob-Storage/index.md

```diff
@@ -124,6 +124,16 @@ You have to manually add `prefix="media/"` to the service element, otherwise Ima
 
 You have now successfully setup Azure Blob Storage with your Umbraco site.
 
+### Application Setting Override
```

Comment:
I'm not sure Override is the correct term for this, how about "Moving settings to Application Settings"?

Reply:
I thought you still need to set the values in the security.config, probably so the service gets added.  

That's why I said override since you need to set it in both places.  But I haven't had the time to try that out to confirm that is true.